### ISSUE TITLE
Suggestions based on the fse package

### ISSUE THREAD

**Issue by avimallu** [kampersanda/sif-embedding#6]
Hi,

I'm excited to see how this package grows - if I may, I'd suggest a glimpse at the FSE package roadmap in https://github.com/oborchers/Fast_Sentence_Embeddings/issues/49 for some additions to the `TODO` things on the README. If a solid Rust base is present for multiple extensions of SIF, it'll make a great base to be used via Python.

Cheers!

**Comment by kampersanda** [comment]
@avimallu 

Thank you for your suggestion and information!

The roadmap will be beneficial in growing this library. I will utilize it for designing the API.

**Comment by avimallu** [comment]
@kampersanda, great progress on the package! Are the plans to build Python bindings still intact?

**Comment by kampersanda** [comment]
@avimallu Yes. But, sif-embedding includes some dependencies to external Rust libraries (at the API level) and I am trying to figure out how to resolve this successfully.

**Comment by kampersanda** [comment]
I created the issue https://github.com/kampersanda/sif-embedding/issues/47
